qa/common/gen_qa_identity_models

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
So the gen_qa_zero_models.py is actually generating identity models. The "zero" name comes from its original use in only the zero testing, but the identity models are now used for much more. So you don't need a whole new file for this... you just need to add identity model for pytorch to existing file (or perhaps as zero file contents to this file so that the naming is better.. "identity" makes more sense than "zero").